robot_smach_states/src/robot_smach_states/designator_iterator.py

```diff
@@ -60,11 +60,17 @@ def __init__(self, collection_des, element_des):
     def execute(self, userdata=None):
         if self._current_elements is None:
             collection = self.collection_des.resolve()
+            if collection is None:
+                rospy.loginfo("Collection is None".format(collection))
```

Comment:
I don't think my suggestion to switch to debug is correct. I am not sure if a `loginfo` here is sufficient. As a `None` shouldn't happen on runtime IMO. An empty collection would be OK for me, but not a None.

Reply:
@LoyVanBeek after some discussion we think you might be the best judge of this. We, however think this should give a warning

Reply:
None is the outcome of a designator if it can't resolve etc. So I'd definitely use `loginfo` (or maybe `logwarn` because it means something in a different place failed earlier). At least don't hide this message by default.

Reply:
The loginfo has been changed to a logwarn